mailpoet/tasks/release/GitHubController.php

```diff
@@ -255,6 +256,43 @@ public function projectBranchExists(string $project, string $branch): bool {
     return true;
   }
 
+  public function mergePullRequest(string $project, string $branch): void {
+    $githubProject = $this->getGithubPathByProject($project);
+    $pullRequest = $this->findPullRequest($githubProject, $branch);
+
+    if (!$pullRequest) {
+      throw new \Exception('Pull Request not found');
+    }
+
+    $response = $this->httpClient->put(
+      "pulls/{$pullRequest['number']}/merge",
```

Comment:
Our repository is configured to allow only the `rebase` merge method. Will the API respect these settings and will use the `rebase` method? Maybe we could specify the method explicitly via [merge_method parameter](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#merge-a-pull-request)

Reply:
Good point. I considered it, but I forgot at the end. Thank you.
I will add it.